focused/security/restFormAuthCustomStore/src/main/webapp/login.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+
+<!--
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose with or without fee is hereby granted.
+
+    THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR(S) DISCLAIMS ALL WARRANTIES
+    WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+    MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR(S) BE LIABLE FOR ANY
+    SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER
+    RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT,
+    NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE
+    USE OR PERFORMANCE OF THIS SOFTWARE.
+
+-->
+
+<html>
+    <head>
+        <title>Login</title>
+    </head>
+    <body>
+        Login to continue
+        <form method="POST" action="j_security_check">
+            <p>
```

Comment:
To go with what @BalusC said, typically you'd use `<div>`s here rather than `<p>`, since these aren't paragraphs in text.